Content-type: text/xml

<?xml version="1.0" encoding="utf-8"?>
<testing-report run-id="49" judge-id="9" status="SE" scoring="ACM" run-tests="3" contest-id="3104" real-time-available="yes" max-memory-used-available="yes" correct-available="yes" failed-test="3" time-limit-ms="2000" real-time-limit-ms="5000" marked-flag="no" >
  <uuid>933e0ce9-49a4-4651-937c-1c723739d239</uuid>
  <host>localhost.localdomain</host>
  <cpu-model>Intel(R) Pentium(R) CPU 3550M @ 2.30GHz</cpu-model>
  <cpu-mhz>2294.738</cpu-mhz>
  <compiler_output></compiler_output>
  <tests>
    <test num="1" status="OK" time="28" real-time="31" max-memory-used="307200"  checker-comment="OK" >
  <program-stats-str>{ utime=27, stime=1, ptime=28, rtime=31, maxvsz=307200, maxrss=6258688, nvcsw=1, nivcsw=8 }</program-stats-str>
  <checker-stats-str>{ utime=0, stime=0, ptime=0, rtime=2, maxvsz=307200, maxrss=2285568, nvcsw=1, nivcsw=1 }</checker-stats-str>
      <input size="2">4
</input>
      <output size="2">7
</output>
      <correct size="2">7
</correct>
      <stderr size="0"></stderr>
      <checker size="3">OK
</checker>
    </test>
    <test num="2" status="OK" time="28" real-time="30" max-memory-used="307200"  checker-comment="OK" >
  <program-stats-str>{ utime=28, stime=0, ptime=28, rtime=30, maxvsz=307200, maxrss=6049792, nvcsw=1, nivcsw=5 }</program-stats-str>
  <checker-stats-str>{ utime=0, stime=0, ptime=0, rtime=1, maxvsz=307200, maxrss=2285568, nvcsw=1, nivcsw=1 }</checker-stats-str>
      <input size="2">5
</input>
      <output size="3">13
</output>
      <correct size="3">13
</correct>
      <stderr size="0"></stderr>
      <checker size="3">OK
</checker>
    </test>
    <test num="3" status="SE" time="286" real-time="319" max-memory-used="18698240" >
  <program-stats-str>{ utime=218, stime=68, ptime=286, rtime=319, maxvsz=18698240, maxrss=19091456, nvcsw=1, nivcsw=40 }</program-stats-str>
      <input size="2">1
</input>
      <output size="0"></output>
      <stderr size="290">Can't open '/usr/share/abrt/conf.d/plugins/python3.conf': Operation not permitted
Can't open '/etc/abrt/plugins/python3.conf': Operation not permitted
Traceback (most recent call last):
  File &quot;./000049&quot;, line 8, in &lt;module&gt;
    stairs[2] = 2
IndexError: list assignment index out of range
</stderr>
    </test>
  </tests>
</testing-report>
